Makefile

```diff
@@ -134,9 +134,10 @@ test-cli:
 	@echo "--> Testing CLI"
 	rm -rf test_cli
 	mkdir test_cli
-	cd test_cli && sentry init test_conf > /dev/null
-	cd test_cli && sentry --config=test_conf upgrade --traceback --noinput > /dev/null
-	cd test_cli && sentry --config=test_conf help 2>&1 | grep start > /dev/null
```

Comment:
Why'd you remove the grep here?

Reply:
It's not very helpful:

```
$ sentry help 2>&1 | grep start
  start        DEPRECATED see `sentry run` instead.
```

I thought we might as well see what all these commands print since it doesn't hurt.